rdkcentral/Lightning-UI-Components
# feat(Badge): add stroke support and clean up docs file

## Description

1. In this PR, I have added stroke functionality to the Badge component, did some cleanup in docs file and fixed the Style properties table by removing extra columns.
2. Added alpha functionality to the Badge component when there is no title or no icon present.

## References

LUI-912

## Testing

1. Navigate to Badge component.
2. Verify that the stroke support has been added.
3. Test the layout and positioning of the title and icon are working as expected
4. Try removing the title in the text sub story and make sure it should be completely alpha out(before it used to have small surface).
